server/src/main/java/io/crate/metadata/RelationName.java

```diff
@@ -159,6 +167,13 @@ public void ensureValidForRelationCreation() throws InvalidSchemaNameException,
         }
     }
 
+    public static void ensureIsCrateCatalog(String catalogName) {
+        if (DB_NAME.equals(catalogName) == false) {
+            throw new IllegalArgumentException(
+                String.format(Locale.ENGLISH, "Valid catalog name is only: [%s]", DB_NAME));
```

Comment:
```suggestion
            throw new IllegalArgumentException(
                String.format(Locale.ENGLISH, "Unexpected catalog name: %s. Only available catalog is %s", catalogName, DB_NAME));
```

Adding the actual value into the error messages can help users spot the mistake.